content/docs/command-reference/exp/show.md

```diff
@@ -0,0 +1,99 @@
+                    [--include-params <params_list>]
+                    [--exclude-params <params_list>]
+                    [--sort-by <metric/param>] [--sort-order {asc,desc}]
+                    [--no-timestamp] [--sha] [--show-json]
+                    [--precision <n>]
+```
+
+## Description
+
+Shows experiments in a detailed table which includes parameters and metrics.
+Only the experiments derived from the Git `HEAD` are shown by default but all
+experiments can be included with the `---all-commits` option. Experiments are
+sorted by timestamp by default.
+
+Your terminal will enter a paginated screen by default, which you can exit by
+typing `Q` in your keyboard. Use `--no-pager` to print the entire table at once
+instead.
+
+```dvc
+$ dvc exp show --no-pager
+┏━━━━━━━━━━━━━━━┳━━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━┓
+┃ Experiment    ┃ avg_prec ┃ roc_auc ┃ train.n_est┃ train.min_split ┃
+┡━━━━━━━━━━━━━━━╇━━━━━━━━━━╇━━━━━━━━━╇━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━┩
+│ workspace     │  0.56191 │ 0.93345 │ 50         │ 2               │
+│ master        │  0.55259 │ 0.91536 │ 50         │ 2               │
+│ ├── exp-bfe64 │  0.57833 │ 0.95555 │ 50         │ 8               │
+│ └── exp-ad5b1 │  0.56191 │ 0.93345 │ 50         │ 2               │
+└───────────────┴──────────┴─────────┴────────────┴─────────────────┘
+```
+
+## Options
+
+- `-a`, `--all-branches` - include experiments derived from the tip of all Git
+  branches instead of just the last commit. Note that this can be combined with
+  `-T` below, for example using the `-aT` flag.
+
+- `-T`, `--all-tags` - same as `-a` above, but applies to Git tags as well as
+  the workspace. Note that both options can be combined, for example using the
+  `-aT` flag.
+
+- `--all-commits` - same as `-a` or `-T` above, but applies to _all_ Git commits
+  as well as the workspace. This prints all experiments available.
+
+- `--no-pager` - do not enter the pager screen. Writes the entire table to
+  standard output. Useful to redirect the output to a file, or use your own
+  paginator.
+
+- `--include-params <list>` - include the specified `dvc params` in the table
+  (only the specified params will be shown). Accepts a comma-separated `list` of
+  param names.
+
+- `--exclude-params <list>` - exclude the specified `dvc params` from the table
+  (all param will be shown except for the specified ones). Accepts a
+  comma-separated `list` of param names.
+
+- `--include-metrics <list>` - include the specified `dvc metrics` in the table
+  (only the specified metrics will be shown). Accepts a comma-separated `list`
+  of metric names.
+
+- `--exclude-metrics <list>` - exclude the specified `dvc metrics` from the
+  table (all param will be shown except for the specified ones). Accepts a
+  comma-separated `list` of metric names.
+
+- `--sort-by <name>` - sort related experiments by the specified metric or param
+  (`name`). Only one sort column (either metric or param) can be specified.
```

Comment:
> related experiments

Is that a hint that only experiments that have the given param/metric will be listed?

Reply:
In this case related means that for a checkpoint run, commits within that checkpoint run will always be grouped together, and that experiments will always be grouped by their original parent commit.

So if you have some experiments that are derived from commit `abc123`, and some experiments that are derived from commit `def456`, the experiments for `abc123` would be sorted and then grouped together, and then the experiments for `def456` would be sorted and grouped together.

Reply:
I see, got it. Thanks

And will (groups) be sorted by the parent's metric/param? Or chronological as usual

```
exp      sort_val
parent1   1
  child1   2
  child2   3
parent3   2
  child2   1
  child1   7
parent2   4
  child1   2
  child2   4
```

Reply:
the top-level parent commits will always be organized chronologically, sorting is only done for (nested) experiment runs

Reply:
Got it. Updated info in 2e263859.